typeform-helpers.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const request = require('request'),
+    _ = require('lodash'),
+    moment = require('moment'),
+    TYPEFORM_TOKEN = require('./api-keys').TYPEFORM_TOKEN,
+    constants = require('./constants'),
+    TYPEFORM_FORM_ID = constants.TYPEFORM_FORM_ID,
+    TYPEFORM_EMAIL_FIELD = constants.TYPEFORM_EMAIL_FIELD,
+    TYPEFORM_TWITTER_FIELD = constants.TYPEFORM_TWITTER_FIELD
+
+function getFromTypeform (params, callback) {
+    // merge params with API key and default completed = true
+    // allow params passed from calling function to overwrite completed
+    const completedParams = { completed: true }
+    const keyParams = { key: TYPEFORM_TOKEN}
+    const fullParams = _.extend({}, completedParams, params, keyParams)
+
+    request({
+        url: 'https://api.typeform.com/v1/form/' + TYPEFORM_FORM_ID,
+        qs: fullParams
+    }, function (error, response, body) {
+        if (callback) {
+            callback(error, response, body)
+        }
+    })
+}
+
+function getLatestResponses (mostRecentDateSubmit, callback) {
+    // convert mostRecentDateSubmit string to Unix timestamp
+    const timestamp = moment.utc(mostRecentDateSubmit).unix()
+    const params = { since: timestamp }
+
```

Comment:
Takes a timestamp in string form, converts it to a Unix timestamp, and requests all responses from Typeform since that time. We'll pass this function the timestamp of the most recent form submission in the database.